README.md

```diff
@@ -94,6 +94,37 @@ Set `absoluteFilePathsForReporter` to `true` to use absolute file paths in gener
 [getting_started]: http://gruntjs.com/getting-started
 [vnujar]: https://validator.github.io/validator/
 
+### `wrapPath`
```

Comment:
This is not a great name. Something a little longer wouldn't hurt, as long as it gets the purpose across a littler better.